examples/expo-web3js/jest.config.js

```
@@ -0,0 +1,48 @@
+/* global __dirname */
+
+const path = require('path');
```

ESLint

[error] A `require()` style import is forbidden.

(`@typescript-eslint/no-require-imports`)

---

[error] 'require' is not defined.

(no-undef)